lost/src/main/java/com/mapzen/android/lost/internal/GeofencingApiImpl.java

```diff
@@ -123,4 +129,16 @@ private void removeGeofences(LostApiClient client, String geofenceRequestId)
     return new SimplePendingResult(hasResult);
   }
 
+  public PendingIntent pendingIntentForIntentId(int intentId) {
+    return idToPendingIntent.get(intentId);
+  }
+
+  public Geofence geofenceForIntentId(int intentId) {
+    return idToGeofence.get(intentId);
+  }
+
+  public void setGeofenceForIntentId(int intentId, Geofence geofence) {
```

Comment:
Rm'd this public method and extracted the id generation into a separate class that implements the new `IdGenerator` interface